py_zipkin/thrift/__init__.py

```diff
@@ -138,6 +138,13 @@ def create_span(
     of the span. Timestamps passed in are in seconds, they're converted to
     microseconds before thrift encoding.
     """
+    # Check if trace_id is 128-bit. If so, record trace_id_high separately.
+    trace_id_length = len(trace_id)
+    trace_id_high = None
+    if trace_id_length > 16:
+        assert trace_id_length == 32
+        trace_id, trace_id_high = trace_id[:16], trace_id[16:]
```

Comment:
I think you have this backwards. The high order bytes are on the left side of the string. See the example here: https://github.com/openzipkin/zipkin/blob/master/zipkin-server/README.md#128-bit-trace-ids